server/presenters/document.js

```diff
@@ -36,6 +36,7 @@ async function present(ctx: Object, document: Document, options: ?Options) {
     team: document.teamId,
     collaborators: [],
     starred: !!(document.starred && document.starred.length),
+    collectionId: document.atlasId,
```

Comment:
why can't we just pull this from `collection`?

Reply:
We could, but it's not reliably available.

Reply:
https://github.com/outline/outline/blob/master/server/presenters/document.js#L44

I don't really see why it wouldn't exist, especially on the backend?

Reply:
On the backend it should be there yep, I guess it's just usually good practice to read from the model rather than the relationship where you can. I'll change it up though.

Reply:
Yeah, I would like to keep the API clean and concise as it will also be exposed publicly